backend/supervisor.d/services.conf

```diff
@@ -101,16 +101,16 @@ startretries=3
 exitcodes=0,2
 stopsignal=INT
 stopwaitsecs=10
-stopasgroup=true
-killasgroup=true
+stopasgroup=false
```

Comment:
Why are these 2 lines changed? What are the implications?

Reply:
Also note that since the configuration of supervisor (in local_configs) is decoupled from version control, we need to alert the users when some change like this happens

Reply:
If the change is not truly required, no need to change the default supervisor configuration.

Reply:
this is needed to avoid having exception when we stop monitors